pkgs/tools/admin/oci-cli/default.nix

```diff
@@ -46,7 +46,8 @@ python3Packages.buildPythonApplication rec {
       --replace "cryptography==3.2.1" "cryptography" \
       --replace "pyOpenSSL==19.1.0" "pyOpenSSL" \
       --replace "PyYAML==5.3.1" "PyYAML" \
-      --replace "six==1.14.0" "six"
+      --replace "six==1.14.0" "six" \
+      --replace "arrow==0.17.0" "arrow"
```

Comment:
To comply with [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#submitting-changes) please have the commit message name be of the format
```
<pkg-name>: <subject-line>
```
for more examples, please look at https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#submitting-changes

in your case, the commit message should be:
```
oci-cli: patch arrow version
```

Reply:
Thanks! Done!